apps/frontend/src/pages/register.tsx

```diff
@@ -227,6 +227,7 @@ const Register: React.FC = () => {
       toast.success("Account created successfully!");
       router.push("/profile");
     } catch (error) {
+      onGoBack();
```

Comment:
This commit addresses specific fixes from https://github.com/cursive-team/connections/issues/19

Reply:
The internal service logging point will be moved into a new issue, it's a larger / separate issue.